include/heuristic/HeuristicMapper.hpp

```diff
@@ -8,6 +8,19 @@
 
 #pragma once
 
+/**
+ * number of two qubit gates acting on pairs of logical qubits in some layer
+ * where the key entry corresponds to logical qubits pairs ({q1, q2}) and
+ * the value entry to the number of gates acting on the pair in each direction
+ * (the first number with control=q1, target=q2 and the second the reverse)
+ *
+ * e.g. with multiplicity {{0,1},{2,3}} there are 2 gates with logical
+ * qubit 0 as control and qubit 1 as target, and 3 gates with 1 as control
+ * and 0 as target
+ */
+using TwoQubitMultiplicity =
+    std::map<Edge, std::pair<std::uint16_t, std::uint16_t>>;
```

Comment:
Is `map` here really preferred over `unordered_map` (=hash map)?

Reply:
Oh, I was not aware, that C++ uses ordered collections by default. Thank you for the suggestion